README.md

```diff
@@ -14,34 +14,50 @@ You need to have postgreSQL installed and set up on your machine.
 
 Clone the repository from [GitHub](https://www.github.com)
 ```
-git clone https://github.com/teamtaverna/taverna_api.git
+$ git clone https://github.com/teamtaverna/taverna_api.git
 ```
 
 ### Installation
 
 **Mac Users**
 
 Be sure to have the following installed and setup first.
+* brew (To install python)
+* Python3
+    ```
+    $ brew install python3
+    ```
 * PostgreSQL (Ensure the server is running)
-* Brew
-* Xcode
-
-Next, step into the project directory and set up your environment by running this script.
-
-```
-./scripts/local_env_setup.sh
-```
-
-**OR follow these steps to set up manually.**
+* Create a database with PostgreSQL. From the terminal you can run
+    ```
+    $ createdb NAME_OF_DB
+    ```
 * Install [Virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/install.html).
+    ```
+    $ pip install virtualenvwrapper
+    ```
+    **You might need to use sudo.** If you have problem with the `six` package during installation, run this instead...
+    ```
+    $ sudo pip install virtualenvwrapper --ignore-installed six
+    ```
+    After installing virtualenvwrapper, add three lines to your shell startup file (.bashrc, .profile, etc.) to set the location where the virtual environments should live, the location of your development project directories, and the location of the script installed with this package:
+    ```
+    export WORKON_HOME=$HOME/.virtualenvs
+    export PROJECT_HOME=$HOME/Devel
+    source /usr/local/bin/virtualenvwrapper.sh
+    ```
+    **After editing it, reload the startup file (e.g., run source ~/.bashrc).**
+
 * Create a virtual environment for the project.
+    ```
+    $ mkvirtualenv NAME_OF_YOUR_ENV -p $(which python3)
```

Comment:
We can install project requirements as part of this step and ensure that when the virtualenv is activated, terminal changes directory to the application directory. Ideally, we would navigate to the project directory, and run

`mkvirtualenv <name_of_env> -a . -r requirements.txt -p $(which python3)`